frontend/src/layout/navigation-3000/navigationLogic.tsx

```diff
@@ -410,6 +410,12 @@ export const navigation3000Logic = kea<navigation3000LogicType>([
                               icon: <IconLive />,
                               to: featureFlags[FEATURE_FLAGS.LIVE_EVENTS] ? urls.activity() : urls.events(),
                           },
+                          {
+                              identifier: Scene.DataWarehouse,
+                              label: 'SQL editor',
```

Comment:
"SQL editor" feels a bit like underselling it. _To me_ it doesn't sound actionable, appears mundane. The "SQL studio" I proposed earlier would feel a bit broader (the point being the ability to _create findings_, not _edit_ them). "SQL access" could be a more technical framing of the _querying_ aspect. I know this is nitpicking a bit – what do you think?